src/satipc.c

```diff
@@ -401,13 +401,14 @@ int satipc_open_device(adapter *ad)
 	if ((sip->last_connect > 0) && (ctime - sip->last_connect < 5000))
 		return 3;
 
+	USockAddr sv;
 	sip->last_connect = ctime;
-	ad->fe = tcp_connect_src(sip->sip, sip->sport, NULL, 1, sip->source_ip[0] ? sip->source_ip : NULL); // blocking socket
+	ad->fe = tcp_connect_src(sip->sip, sip->sport, &sv, 1, sip->source_ip[0] ? sip->source_ip : NULL); // blocking socket
 	if (ad->fe < 0)
 		LOG_AND_RETURN(2, "could not connect to %s:%d", sip->sip, sip->sport);
 
-	LOG("satipc: connected to SAT>IP server %s port %d %s handle %d %s %s", sip->sip,
-		sip->sport, sip->use_tcp ? "[RTSP OVER TCP]" : "", ad->fe, sip->source_ip[0] ? "from source" : "", sip->source_ip[0] ? sip->source_ip : "");
+	LOG("satipc: connected to SAT>IP server %s port %d%s (socket handle %d) %s %s", sip->sip,
```

Comment:
Do jot add socket. There is sockets (struct socket in socketworks) and handle (unix handle). This will be confusing

Reply:
Hi,

To be true, sometimes I don't fully understand the original messages. However, I reviewed all the code and I've used the same names every time:

- `Handle` is the sock number (unix handle).
- `Socket` is the minisatip socket id (struct socket).
- `sid` is the sid of the connection.

I'm using _consistent_ names in the entire patch. However, in this case I'm not sure if `ad->fe` represents "socket" or "handle". What you prefer?

Reply:
ad->fe is none of them, we can leave it as it is... or add ad->fe %d